calyx-py/test/invoke-with-builder.py

```diff
@@ -0,0 +1,53 @@
+import calyx.builder as cb
```

Comment:
I am curious: what is this file testing? What does this test have to do with the source loc feature you are adding in this PR?

Reply:
This was supposed to test generating a source info block and `pos` annotations, and I've moved the test to `calyx-py/test/source-loc/invoke-with-builder.py` to reflect this!